src/schemas/json/bxci.schema-2.x.json

```diff
@@ -151,9 +151,22 @@
         "publish": {
           "$ref": "#/definitions/outputDockerPublish",
           "description": "Docker publishing information. If not present, no images will be published"
+        },
+        "vulnerability_scan": {
+          "type": "object",
```

Comment:
Optional fix: `title` property along with `description` can be added to enable VS Code editor properly name incorrectly passed value to `vulnerability_scan`.

Reply:
Thank you. Added!